harness/tests/determined/common/experimental/test_workspace.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import responses
+
+from determined.common import api
+from determined.common.api import bindings
+from determined.common.experimental import workspace
+from tests.fixtures import api_responses
+
+_MASTER = "http://localhost:8080"
+
+
+@pytest.fixture
+def standard_session() -> api.Session:
```

Comment:
not really a comment for this PR, but i wonder if these common fixtures like `standard_session` and `_MASTER` should be defined elsewhere, like in a shared utils or something. it seems like we're using them for every test, so it's a bit of duplicated code. not a big deal, and not targeted specifically at this PR. just a thought.